Build/Build.cs

```diff
@@ -75,6 +76,9 @@ class Build : NukeBuild
         .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
         .Executes(() =>
         {
+            Information(BranchSpec);
```

Comment:
🤔 What does this have to do with the `Clean` target?

Reply:
Nothing really.. I am unsure wether to keep or remove it.. I will remove this for now